2021-04-27T20:53:06.8555848Z ##[group]Run python -m pip install --upgrade pip wheel
2021-04-27T20:53:06.8556769Z [36;1mpython -m pip install --upgrade pip wheel[0m
2021-04-27T20:53:06.8557810Z [36;1mpython -m pip install -r requirements.txt[0m
2021-04-27T20:53:06.8610503Z shell: /usr/bin/bash -e {0}
2021-04-27T20:53:06.8611002Z env:
2021-04-27T20:53:06.8611610Z   pythonLocation: /opt/hostedtoolcache/Python/3.9.4/x64
2021-04-27T20:53:06.8612445Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.4/x64/lib
2021-04-27T20:53:06.8613233Z ##[endgroup]
2021-04-27T20:53:10.5219430Z WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
2021-04-27T20:53:10.5222019Z distutils: /opt/hostedtoolcache/Python/3.9.4/x64/include/python3.9/UNKNOWN
2021-04-27T20:53:10.5224288Z sysconfig: /opt/hostedtoolcache/Python/3.9.4/x64/include/python3.9
2021-04-27T20:53:10.5227464Z WARNING: Additional context:
2021-04-27T20:53:10.5228824Z user = False
2021-04-27T20:53:10.5229986Z home = None
2021-04-27T20:53:10.5231217Z root = None
2021-04-27T20:53:10.5232354Z prefix = None
2021-04-27T20:53:10.6282634Z Requirement already satisfied: pip in /opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages (21.1)
2021-04-27T20:53:10.9185854Z Collecting wheel
2021-04-27T20:53:10.9560448Z   Downloading wheel-0.36.2-py2.py3-none-any.whl (35 kB)
2021-04-27T20:53:11.0325543Z Installing collected packages: wheel
2021-04-27T20:53:11.0731552Z WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
2021-04-27T20:53:11.0733218Z distutils: /opt/hostedtoolcache/Python/3.9.4/x64/include/python3.9/UNKNOWN
2021-04-27T20:53:11.0734323Z sysconfig: /opt/hostedtoolcache/Python/3.9.4/x64/include/python3.9
2021-04-27T20:53:11.0763607Z WARNING: Additional context:
2021-04-27T20:53:11.0764956Z Successfully installed wheel-0.36.2
2021-04-27T20:53:11.0766526Z user = False
2021-04-27T20:53:11.0767389Z home = None
2021-04-27T20:53:11.0768692Z root = None
2021-04-27T20:53:11.0769594Z prefix = None
2021-04-27T20:53:11.6163931Z WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
2021-04-27T20:53:11.6165771Z distutils: /opt/hostedtoolcache/Python/3.9.4/x64/include/python3.9/UNKNOWN
2021-04-27T20:53:11.6168037Z sysconfig: /opt/hostedtoolcache/Python/3.9.4/x64/include/python3.9
2021-04-27T20:53:11.6168974Z WARNING: Additional context:
2021-04-27T20:53:11.6169600Z user = False
2021-04-27T20:53:11.6170122Z home = None
2021-04-27T20:53:11.6171059Z root = None
2021-04-27T20:53:11.6171762Z prefix = None
2021-04-27T20:53:11.7039618Z ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
2021-04-27T20:53:11.7868313Z ##[error]Process completed with exit code 1.
